app/src/forms/email/emailService.js

```diff
@@ -200,13 +210,18 @@ const service = {
         subject: `${form.name} Accepted`,
         priority: 'normal',
         messageLinkText: `Thank you for your ${form.name} submission. You can view your submission details by visiting the following links:`,
+        body,
         form,
       };
       return service._sendEmailTemplate('sendSubmissionConfirmation', configData, submission, referer);
     } catch (e) {
-      log.error(`formId: ${formId}, submissionId: ${submissionId}, referer: ${referer}`, { function: 'submissionReceived' });
-      log.error(e.message, { function: 'submissionConfirmation' });
-      log.error(e);
+      log.error(e.message, e, {
+        function: 'submissionConfirmation',
+        formId: formId,
+        submissionId: submissionId,
+        body: JSON.stringify(body),
```

Comment:
It may be fine to leave the body as a JSON object directly, as stringifying it will likely add a bunch of character encoding wrappers on the content. Winston can handle direct object references while logging in JSON mode.

Reply:
removing the JSON.stringify call on the body